analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.LearnConstraint.cs

```diff
@@ -1236,6 +1236,136 @@ public void Branching_LearnsNumberConstraint_FromRanges_HasNumber(string express
         validator.ValidateTag("Arg", x => x.Should().HaveOnlyConstraints(ObjectConstraint.NotNull, NumberConstraint.From(expectedMin, expectedMax)));
     }
 
+    [DataTestMethod]
+    [DataRow("left >=  0", 0, null)]
+    [DataRow("right >= 0", 0, null)]
+    [DataRow("left >=  0 && right <= 50", 0, 50)]
+    [DataRow("left <= 50 && right >=  0", 0, 50)]
+    [DataRow("left >=  0 && left <= 50 && right >= 10 && right <= 20", 10, 20)]
+    [DataRow("left >= 10 && left <= 20 && right >=  0 && right <= 50", 10, 20)]
+    [DataRow("left >=  0 && left <= 20 && right >= 10 && right <= 50", 10, 20)]
+    [DataRow("left >= 10 && left <= 50 && right >=  0 && right <= 20", 10, 20)]
+    public void Branching_LearnsNumberConstraint_FromRanges_Equals(string expression, int? expectedMin, int? expectedMax)
```

Comment:
Add two missing tests and then reformat so that the operators are aligned. Do the same for the tests following below.
```
    [DataRow("left  <= 50", null, 50)]
    [DataRow("right <= 50", null, 50)]
```